### PR TITLE
Add Code of Conduct

### PR DESCRIPTION
**Pull request by prertik**
@Mouvedia is this okay?

Fix: #2375

### CODE REVIEW

**Comment by Mouvedia** [comment]
You are the judge.

Id prefer something short and simple like I proposed in #2375 but ultimately it's purely symbolical: your first commit needs to be welcoming and positive.

**Comment by prertik** [comment]
I used [conduct](https://github.com/sindresorhus/conduct) to generate this CoC. I believe it's more elaborate.

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+threatening, offensive, or harmful.
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces
+when an individual is representing the project or its community. Examples of
+representing a project or community include using an official project e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event. Representation of a project may be
+further defined and clarified by project maintainers.
```
Comment (Mouvedia) [inline CODE-OF-CONDUCT.md]: That's a big loose.

Comment (prertik) [reply CODE-OF-CONDUCT.md]: What do you propose?

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the project team leader at prertik@outlook.com. All
```
Comment (Mouvedia) [inline CODE-OF-CONDUCT.md]: Do you own prertik@lighttable.com?

Comment (prertik) [reply CODE-OF-CONDUCT.md]: Nope. So, my email id.

File: CODE-OF-CONDUCT.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
```
Comment (Mouvedia) [inline CODE-OF-CONDUCT.md]: "repercussions" is quite vague

File: CODE-OF-CONDUCT.md (hunk 1 above)
Comment (Mouvedia) [reply CODE-OF-CONDUCT.md]: It's a good thing, let's move on.

**Comment by prertik** [comment]
So, @Mouvedia what do we do? Go with shorter version? As you proposed...

**Pull request merged**